exs/mundo_1/javascript/035.js

```diff
@@ -0,0 +1,17 @@
+const input = require('readline-sync').question //Chamando o modulo readline-sync para entrada de dados
+
+//Entrada de dados
+let a = input("Primeiro segmento de reta: \n")
+let b = input("Segundo segmento de reta: \n")
+let c = input("Terceiro segmento de reta: \n")
+
+//Condição if e else para saber se as partes formam um triangulo
```

Comment:
Correção ortográfica necessária

```suggestion
//Condição if e else para saber se as partes formam um triângulo
```